### ISSUE TITLE
Dogs?

### ISSUE THREAD

**Issue by skade** [RustFestEU/zurich.rustfest.eu#21]
https://twitter.com/RustFest/status/885134727627378689

There was the question if dogs are allowed at the conference.

There's multiple questions here:
- Does the venue allow animals at all?
- How do we handle comfort animals (this is an accessibility issue: https://en.wikipedia.org/wiki/Emotional_support_animal)? That's not in the original question, but once we're here, let's answer that one, too.
- How do we arrange so that people with fear of dogs//allergies can easily stay out of the way?

I'm generally _pro_, if there's good reason for the animal to be around. I wouldn't want to have everyone bring their animal (just for management reasons), but I'd be happy to accommodate those that need to bring them.

**Comment by oli-obk** [comment]
I have a related suggestion: If the number of emotional support animals is below 1% of the number of participants, allow filling the "free slots" up to the 1% by ballot. Just to raise awareness and display the lack of issues with animals at conferences. The 1% is random, but 3 animals on a 300 ppl conference sounds manageable.

> How do we arrange so that people with fear of dogs//allergies can easily stay out of the way?

At the CCCongress dogs were handled in the way that the holder needs to keep the dog away from any other participants, except if explicitly asked. In smaller events the dog owners were informed who's allergic or phobic in order to prevent any issues from coming up by staying well away.

So basically the oppsite of the quoted question: How do we arrange it so that dogs stay out of the way of people with fear of dogs//allergies?

**Comment by Hoverbear** [comment]
I support RuffFest in general, but we need to bark at the venue before we start wagging our tails to hard. I don't want to get any of us tossed in the doghouse about this.

**Comment by dns2utf8** [comment]
So we have a reply from the venue: No, unless the dog is a guide dog.
```
Allgemeinen Ordnungsvorschriften 4.6: „Tiere sind in den Gebäuden nicht gestattet. Ausnahmen sind das Mitführen von Blinden- und Diensthunden.“
```

**Comment by oli-obk** [comment]
:( ok, well it was worth a try.

**Comment by skade** [comment]
Clarified through #30, a bit unhappy to not be of service there :(.
